### ISSUE TITLE
aria-hidden-no-focusable-content-6cfa84 passed example 3 causes accessibility problems

### ISSUE THREAD

**Issue by dd8** [act-rules/act-rules.github.io#1724]
`tabindex=-1` makes an element unfocusable with the keyboard, but it can still be activated by clicking and touch
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

On an iPad the button in Passed Example 3 can be activated by touch when VoiceOver is off. When VoiceOver is switched on the button can no longer be activated.
https://act-rules.github.io/testcases/6cfa84/0567a6c937cc508606af0ab2854d8a7d059ea2e2.html

The same problem happens on a Mac with Safari and VoiceOver - with VoiceOver off the button can be activated by clicking. With VoiceOver on the page is voiced as "page has no content" and the button cannot be activated.

This example causes the same accessibility problem as a clickable div with no role or tabindex.

Spotted this when reviewing https://github.com/act-rules/act-rules.github.io/pull/1719
